flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -1483,6 +1483,7 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
             const message = event.data;
             if (message.startsWith("event:")) {
                 this.onEvent(`1_BTN_${message.substring(6)}`);
+                SimVar.SetSimVarValue(`L:A32NX_MCDU_PUSH_ANIM_1_${message.substring(6)}`, "Number", 1);
```

Comment:
```suggestion
                SimVar.SetSimVarValue(`H:A320_Neo_CDU_1_BTN_${message.substring(6)}`, "number", 0);
                SimVar.SetSimVarValue(`L:A32NX_MCDU_PUSH_ANIM_1_${message.substring(6)}`, "Number", 1);
```

Remove the call to onEvent as this will be done by firing the H: event. 

This should now work for YourControls and also for the MCDU Server as far as I tested it.